package.json

```diff
@@ -17,6 +17,7 @@
     "less": "^3.9.0",
     "less-loader": "^5.0.0",
     "lodash": "^4.17.15",
+    "moment": "^2.29.1",
```

Comment:
Moment.js is deprecated (see [here](https://momentjs.com/docs/#/-project-status/)) and it's probably best to use other alternatives for libraries, since who knows what will happen. You may want to try:
- https://moment.github.io/luxon/ (straight from the same people)
- https://day.js.org/

Reply:
The problem is that Ant Design `v3.x`, which we currently use, has Moment.js as a built-in dependency, so switching to using a different date package wouldn't have any effect. If there ends up being an issue, we can upgrade to Ant Design 4.0 during the redesign and make the switch.

Reply:
Got it. Will make an issue for this.